app/js/FileController.js

```diff
@@ -61,18 +61,18 @@ function getFile(req, res, next) {
     }
 
     pipeline(fileStream, res, err => {
-      if (err) {
+      if (!fileStream.destroyed) {
```

Comment:
This was not moved out of the `if (err ...)` branch, was that intentional?

Reply:
Yes. I figured that if we can be sure not to destroy things unnecessarily we may as well check it even if there hasn't been an error.